.github/workflows/schedule.yml

```diff
@@ -0,0 +1,151 @@
+name: SCHEDULED
+on:
+  workflow_dispatch:
+  schedule:
+    #  Exectues e2e compatibility test every Wednesday at 1:00 AM and if you need to convert to UTC+8 timezone, you need to -8 hours for cron expression
+    - cron: "0 17 * * 2"
+
+env:
+  CONTAINER_RUN_OPTIONS: " "
+  GINKGO_VERSION: "v1.16.4"
+
+jobs:
+  lint:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    name: lint
+    env:
+      GOPATH: /home/runner/work/${{ github.repository }}
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          path: ./src/github.com/${{ github.repository }}
+
+      - uses: actions/cache@v3
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+      - name: Install dependences
+        run: |
+          sudo apt-get install -y jq
+
+  image-prepare:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    name: Prepare kubeedge/build-tools image
+    steps:
+      - name: Pull kubeedge/build-tools image
+        run: |
+          docker pull kubeedge/build-tools:1.19.12-ke2
+          mkdir -p /home/runner/build-tools/
+          docker save kubeedge/build-tools:1.19.12-ke2 > /home/runner/build-tools/build-tools.tar
+
+      - name: Temporarily save kubeedge/build-tools image
+        uses: actions/upload-artifact@v3
+        with:
+          name: build-tools-docker-artifact
+          path: /home/runner/build-tools
+
+  build:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    name: Multiple build
+    needs: image-prepare
+    steps:
+      - uses: actions/cache@v3
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Retrieve saved kubeedge/build-tools image
+        uses: actions/download-artifact@v3
+        with:
+          name: build-tools-docker-artifact
+          path: /home/runner/build-tools
+
+      - name: docker load kubeedge/build-tools image
+        run: |
+          docker load < /home/runner/build-tools/build-tools.tar
+
+      - run: make
+
+      - run: make smallbuild
+
+      - run: make crossbuild
+
+      - run: make crossbuild ARM_VERSION=GOARM7
+
+      - run: make crossbuild WHAT=cloudcore ARM_VERSION=GOARM8
+
+  k8s_compatibility_schedule_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        protocol:
+          - WebSocket
+          - QUIC
+        version:
+          - v1.24.7
+          - v1.23.13
+          - v1.22.15
+          - v1.21.14
+          - v1.20.15
```

Comment:
Since the K8s dependency vesion has been updated to 1.26, we need to update the version here, from 1.22~1.26?

@wlq1212 @Shelley-BaoYue

Reply:
version has been updated to 1.22~1.26
![image](https://github.com/kubeedge/kubeedge/assets/95007522/8bf39859-ca0b-4c85-8eff-b558205e9c4c)